documentation/cobigen-core_configuration.asciidoc

```diff
@@ -378,6 +378,119 @@ However, if we want to reference an increment that it is not defined inside our
 
 The ref string is split using as delimiter `::`. The first part of the string, is the `trigger_id` to reference. That trigger contains an `increment_id`. Currently, this functionality only works when both templates use the same kind of input file.
 
+== Configuration Upgrade
+
+`CobiGen 2021.12.007` introduces an updated configuration folder structure:
+
+```
+template-sets
+ |- downloaded
+   |- template-set1.jar
+   |- template-set2.jar
+   |- template-set-utilities.jar
+ |- adapted
+   |- template-set1
+     |- src
+       |- main
+         |- templates
+           |- templates
+             |- context.xml
+             |- templates.xml
```

Comment:
I think we should just use the template-set.xml introduced in #1495 here as this configuration (context and templates.xml in the same folder) will not be deployed.

Reply:
Done